src/app/components/CurrentConditionsCard.tsx

```diff
@@ -0,0 +1,44 @@
+import dayFromDate from "../util/dayFromDate";
+import { formatTime } from "../util/formatTime";
+import CardBorder from "./CardBorder";
+import { weatherConditionIcons, WeatherIconKey } from "./WeatherConditionIcons";
+export interface CurrentConditionsCardProps {
+  date: string;
+  time: string;
+  currentTemp: number;
+  feelslike: number;
+  conditions: string;
+  icon: WeatherIconKey;
+  resolvedAddress: string;
+  tempmax: number;
+  tempmin: number;
+}
+
+export default function CurrentConditionsCard(
+  props: CurrentConditionsCardProps
+) {
+  const WeatherIcon = weatherConditionIcons[props.icon];
+  return (
+    <CardBorder>
+      <div className="flex justify-between">
+        <span>{props.resolvedAddress}</span>
+        <span>
+          As of {formatTime(props.time)}, {dayFromDate(props.date)}
+        </span>
+      </div>
+      <div>
+        {props.currentTemp}°{" "}
```

Comment:
```suggestion
        {props.currentTemp}°
```
that space isn't needed, right?